app/models/label/LabelTable.scala

```diff
@@ -765,7 +782,7 @@ object LabelTable {
     * @return LabelMetadata object
     */
   def labelAndTagsToLabelMetadata(label: (Int, String, Boolean, String, Float, Float, Int, Int, Int, Int, Int, Int, String, String, Option[java.sql.Timestamp], String, String, Option[Int], Boolean, Option[String]), tags: List[String]): LabelMetadata = {
-      LabelMetadata(label._1, label._2, label._3, label._4, label._5, label._6, label._7, label._8,
```

Comment:
I agree with this fix. But while you're here, can you fix up the rest of the formatting? I think it would be better to just have:
```
    LabelMetadata(
      label._1, label._2, label._3, label._4, label._5, label._6, label._7, label._8, label._9, label._10, label._11,
      label._12,label._13,label._14,label._15,label._16, label._17, label._18, label._19, label._20, tags
    )
```

Then can you have the similar function you wrote have that same formatting?